include/z64scene.h

```diff
@@ -495,24 +497,24 @@ typedef struct {
 } Path; // size = 0x8
 
 typedef struct {
-    /* 0x0 */ UNK_TYPE2 unk0;
-    /* 0x2 */ UNK_TYPE2 unk2;
-    /* 0x4 */ UNK_TYPE2 unk4;
-    /* 0x6 */ UNK_TYPE2 unk6;
-    /* 0x8 */ UNK_TYPE2 unk8;
+    /* 0x00 */ u16 mapId;
+    /* 0x02 */ s16 unk2;
+    /* 0x04 */ s16 unk4;
+    /* 0x06 */ s16 unk6;
+    /* 0x08 */ u16 unk8; // flags; 1 = mirror x? 2 = mirror y?
```

Comment:
```suggestion
    /* 0x0 */ u16 mapId;
    /* 0x2 */ s16 unk2;
    /* 0x4 */ s16 unk4;
    /* 0x6 */ s16 unk6;
    /* 0x8 */ u16 unk8; // flags; 1 = mirror x? 2 = mirror y?
```
A lot of leading 0's were added in `z64scene.h`, could you revert those?